repos/spack_repo/builtin/packages/flecsi/package.py

```diff
@@ -125,6 +125,11 @@ class Flecsi(CMakePackage, CudaPackage, ROCmPackage):
     conflicts("+hdf5", when="backend=hpx", msg="HPX backend doesn't support HDF5")
     conflicts("^hpx networking=none", when="backend=hpx")
 
+    for cxxstd in ("11", "14"):
+        conflicts(f"^kokkos cxxstd={cxxstd}")
```

Comment:
Can `conflicts("^kokkos cxxstd=11")` be a problem if (some version of) Kokkos doesn't have that variant at all, or is that considered irrelevant because it's `conflicts`?